example_omitempty_test.go

```diff
@@ -0,0 +1,59 @@
+package sqlair_test
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/canonical/sqlair"
+)
+
+func Example() {
+	type Person struct {
+		ID         int    `db:"id,omitempty"`
+		Fullname   string `db:"name"`
+		PostalCode int    `db:"postal_code"`
+	}
+
+	fred := Person{Fullname: "Fred", PostalCode: 1000}
+	mark := Person{Fullname: "Mark", PostalCode: 1500}
+	mary := Person{Fullname: "Mary", PostalCode: 3500}
+	dave := Person{Fullname: "James", PostalCode: 4500}
+	allPeople := []Person{fred, mark, mary, dave}
+
+	sqldb, err := sql.Open("sqlite3", ":memory:")
```

Comment:
It would be best to use the full string found in `package.go` tests here rather then ":memory:" so that we are consistent across the codebase.

Reply:
Synced offline, we will use `file:exampleOmitTest.db?cache=shared&mode=memory` to prevent errors. When go database.sql internally spawns a new connection, `:memory:` will create a totally new db resulting in errors such as "no table found".